faq/post_proof.php

```diff
@@ -44,6 +44,7 @@
         <li><a href="#markup">Check the markup</a></li>
         <li><a href="#title">Straighten up the title page, table of contents,
             and list of illustrations</a></li>
+        <li><a href="#covers">Cover image</a></li> // dummy ID to provide path to wiki FAQ
```

Comment:
You don't need any of the changes in this file.

Reply:
Fixed.